tsl/test/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ add_subdirectory(ssl)
 set(_local_install_checks)
 set(_install_checks)
 
+find_program(PROVE prove
```

Comment:
Looks like this should be done as `PG_REGRESS` below. To support case if prove is not found

```
if (PROVE)
  add_custom_target(provecheck ...)
  list(APPEND _install_checks provecheck)
endif()
```

Btw why do we need to specify path here? Will the prove be found, if installed in `/usr/local/bin` for example?

Reply:
Hmm, `prove` is a basic perl utility. So it should be present really :-)

The path to `find_program` is a HINT. It will search in all other basic system paths as well.

Reply:
I do not think this is critical, but here's the description about using `HINTS` from the manual:

> Search the paths specified by the HINTS option. These should be paths computed by system introspection, such as a hint provided by the location of another item already found. Hard-coded guesses should be specified with the PATHS option.